helm/charts/hpe-csi-driver/crds/hpe-snapshotgroupinfo-crd.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
could you look into making this `v1` instead of `v1beta1`? We have made the changes to other crds including hpenodeinfo to be v1

Reply:
is this specific to 1.20 K8S version change , whether it will work for 1.16 and below version?
and upgrade scenario
From the below link 
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#field-pruning
I

Reply:
We support 1.17+ with CSI 1.4 so this should work with all our supported verisons in csi 1.4.